packages/open_aea/connections/p2p_libp2p/libp2p_node/utils/utils.go

```diff
@@ -250,8 +252,19 @@ func FetchAIPublicKeyFromPubKey(publicKey crypto.PubKey) (string, error) {
 	return hex.EncodeToString(raw), nil
 }
 
+// EthereumPublicKeyFromPubKey return Ethereum's format serialized public key
+func EthereumPublicKeyFromPubKey(publicKey crypto.PubKey) (string, error) {
+	raw, err := publicKey.Raw()
+	if err != nil {
+		return "", err
+	}
+	// return hexutil.Encode(raw), nil
+	return "0x" + hex.EncodeToString(raw)[2:], nil
```

Comment:
what's hex.EncodeToString(raw)[:2]? why are we dropping it?

Reply:
these are some constant characters that, as i understand it, reference the elliptic curve that was used. 
https://davidederosa.com/basic-blockchain-programming/elliptic-curve-keys/

Reply:
Probably add a note in code - it's non-obvious and should be explained there for posterity